geode/google/client.py

```diff
@@ -37,33 +69,42 @@ class Client(m.distance_matrix.Client, m.geocoding.Client):
             )
         )
 
-    async def geocode(self, location: m.Location, session=None) -> Sequence[m.geocoding.Result]:
+    async def _geocode(self, location: m.Location, session=None) -> GoogleGeocodingResponse:
```

Comment:
separate out private method for call.
this is the portion to be retried on failure